frontend/test/metabase-smoketest/README.md

```diff
@@ -6,6 +6,8 @@ These files exist to ensure that we can always do the things we expect with Meta
 
 From the root of the repository:
 
+- If you are running tests that include admin_setup, run `python -m smtpd -n -c DebuggingServer localhost:1025`in terminal first for setting up email through your localhost
```

Comment:
Oh this is handy - we'll need to set something up in our CI environment to do similar. Until we do that, we should have the test intentionally fail.